src/sweeps/params.py

```diff
@@ -118,6 +119,12 @@ def cdf(self, x: ArrayLike) -> ArrayLike:
             return stats.uniform.cdf(
                 np.log(x), self.config["min"], self.config["max"] - self.config["min"]
             )
+        elif self.type == HyperParameter.INV_LOG_UNIFORM:
+            return 1 - stats.uniform.cdf(
+                np.log(1 / x),
+                1 / np.exp(self.config["max"]),
```

Comment:
why max here?

Reply:
min and max are now in log(1/x) space, so these scale and shape parameters should be self.config["min"] and self.config["max"] respectively